opm/parser/eclipse/CMakeLists.txt

```diff
@@ -287,9 +288,12 @@ EclipseState/Tables/ColumnSchema.hpp
 EclipseState/Tables/TableEnums.hpp
 EclipseState/Tables/TableSchema.hpp
 EclipseState/Tables/TableIndex.hpp
-Utility/Functional.hpp)
+#
+Utility/Functional.hpp
+Utility/Stringview.hpp
+Utility/EquilWrapper.hpp)
```

Comment:
Not really part of this PR; but can you move `EquilWrapper` to `EclipseState/InitConfig`?